dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -779,6 +785,54 @@ ERROR_SINGLE_REPO_CONNECTION_FAILED=CWWKF1389E: Unable to create a JSON file at
 ERROR_SINGLE_REPO_CONNECTION_FAILED.explanation=The feature in the ESA file will not be installed.
 ERROR_SINGLE_REPO_CONNECTION_FAILED.useraction=Ensure that write access is available for the directory where the JSON file will be created.
 
+ERROR_FAILED_TO_CONNECT_MAVEN=CWWKF1390E: The configured Maven Repository cannot be reached. Verify that your computer has network access and firewalls are configured correctly, then try the action again. If the connection still fails, the repository server might be temporarily unavailable.
+ERROR_FAILED_TO_CONNECT_MAVEN.explanation=An error occurred when connecting to the configured Maven Repository.
+ERROR_FAILED_TO_CONNECT_MAVEN.useraction=Ensure that the system can access the internet and the appropriate repository.
+
+ERROR_CHECKSUM_FAILED_MAVEN=CWWKF1391E: Failed to validate available checksums for file: {0}
+ERROR_CHECKSUM_FAILED_MAVEN.explanation=The checksum of the downloaded asset is not correct. The content of the downloaded asset is not the same as the content from the repository, so the asset cannot be installed.
+ERROR_CHECKSUM_FAILED_MAVEN.useraction=Run the command again. If the problem continues, see the problem determination information on the WebSphere Application Server Support web page: http://www.ibm.com/software/webservers/appserv/was/support.
+
+ERROR_INVALID_MAVEN_CREDENTIALS=CWWKF1392E:Incorrect credentials provided for the following repository: {0}
+ERROR_INVALID_MAVEN_CREDENTIALS.explanation=Failed to authenticate with the configured repository.
+ERROR_INVALID_MAVEN_CREDENTIALS.useraction=Please supply the correct credentials for the configured repository.
+
+ERROR_FAILED_TO_DOWNLOAD_ASSETS_FROM_MAVEN_REPO=CWWKF1393E:Failed to download artifacts from the configured repository.
```

Comment:
Change
"Failed to download artifacts from the configured repository."
To
"The artifacts from the configured repository failed to download."